back-end/package.json

```diff
@@ -5,9 +5,9 @@
   "scripts": {
     "server": "node server.js",
     "dev": "nodemon server",
-    "lint": "eslint -c .eslintrc.json --ext .js .",
+    "lint": "eslint -c .eslintrc.json --ext .js . --fix",
```

Comment:
Adding back that `--fix` option breaks the pre-commit git hook. See [this PR setting up husky](#13) for details.

If you want to fix the ESLint issues, you could use `npm run link:fix`

Reply:
Sorry about that! I'll fix it tmrw first thing.